### PR TITLE
All the work so far

### PR DESCRIPTION
**Pull request by grae22**

### CODE REVIEW

File: McsaMeetsMailer/BusinessLogic/FullScheduleEmailBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿using McsaMeetsMailer.Models;
+using McsaMeetsMailer.Utils.Html;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace McsaMeetsMailer.BusinessLogic
+{
+  public static class FullScheduleEmailBuilder
+  {
+    public static string Build(IEnumerable<MeetDetailsModel> meetDetails, IHtmlBuilder htmlBuilder)
```
Comment (grae22) [inline McsaMeetsMailer/BusinessLogic/FullScheduleEmailBuilder.cs:10]: Will ultimately need to order columns as per the original sheet.

File: McsaMeetsMailer/Utils/Html/HtmlBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+      foreach (var value in values)
+      {
+        _htmlBuilder.Append($"<td>{value}</td>");
+      }
+
+      _htmlBuilder.Append("</tr>");
+    }
+
+    public string GetHtml()
```
Comment (grae22) [inline McsaMeetsMailer/Utils/Html/HtmlBuilder.cs:49]: May be possible to perform some validation at this point - e.g. if you track the number of "open" table elements then at this point that count should be zero.

**Review by melzatkinson — APPROVED** [review]

File: McsaMeetsMailer/Utils/Html/HtmlBuilder.cs (hunk 2 above)
Comment (melzatkinson) [reply McsaMeetsMailer/Utils/Html/HtmlBuilder.cs:49]: Added ticket

File: McsaMeetsMailer/BusinessLogic/FullScheduleEmailBuilder.cs (hunk 1 above)
Comment (melzatkinson) [reply McsaMeetsMailer/BusinessLogic/FullScheduleEmailBuilder.cs:10]: Added ticket to rethink use of dictionary.

**Pull request merged**